cluster/terraform/main.tf

```diff
@@ -158,7 +158,7 @@ module "eks_blueprints_addons" {
   enable_karpenter = true
 
   karpenter = {
-    chart_version       = "0.37.0"
+    chart_version       = "1.0.0"
```

Comment:
A new release came out that is fixing a problem if you're using Graviton to run Karpenter. Please change it to `1.0.1` (here and in the README)